sources/packages/backend/apps/db-migrations/src/migrations/1671491858132-RemoveATBCIntegrationQueue.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class RemoveATBCIntegrationQueue1671491858132
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Remove-atbc-integration-queue.sql", "Queue"),
```

Comment:
mm... why are we removing atbc queue configuration?

Reply:
The apply for PD status call to ATBC API is retained in API for the purpose that when a student is applying for PD from application, and once the ATBC API is called the Apply PD button needs to be disabled right away.

Hence removing the queue atbc-integration and it's queue configuration.

Reply:
so, are we not adding it to the queue?  it was added as per the decision with the Schwenker

Reply:
Yes we are not adding it to queue. I'm sure that we didn't have this scenario in mind that time. As discussed, when a student is applying for PD, there must be an instant response coming from source of truth confirming that they have applied for the PD and hence the apply for PD button must get disabled.

Button being enabled there after applying for PD is not a standard behavior. 

As of now I think  this is the only situation, where we need a response to update UI while calling an external system. Considering to put a solution in place only for this to be used through queues is required or not? even if required it's scope doesn't belong to story and hence retained it's current behavior of calling ATBC from API itself after discussion with @andrewsignori-aot .